test/table.ts

```diff
@@ -95,7 +95,7 @@ describe('Bigtable/Table', function() {
   let TABLE_NAME;
 
   let Table: typeof tblTypes.Table;
-  let table: tblTypes.Table;
+  let table: any;
```

Comment:
don't we need eslint-ignore here? how does it work?

Reply:
Yikes, good call out.  For some reason we don't...  will fix this after we have user land types working properly (more important that tests)